tests/system/feature_store/test_feature_store.py

```diff
@@ -592,6 +592,36 @@ def test_nosql_no_path(self):
             1
         ].get_path().get_absolute_path() == fset.get_target_path("nosql")
 
+    @pytest.mark.parametrize("local", [True, False])
+    def test_ingest_with_format_run_project(self, local):
+        first_data = pd.read_csv("/Users/Tomer_Mamia/Downloads/first_data.csv")
+        feature_set_1 = fstore.FeatureSet(
+            name=f"fs-run_project-format-local-{local}",
+            entities=["entity"],
+            timestamp_key="time",
+        )
+        artifact_path = mlrun.mlconf.artifact_path
+        targets = [
+            CSVTarget(name="labels", path=os.path.join(artifact_path, "file.csv"))
+        ]
+        feature_set_1.set_targets(targets=targets, with_defaults=False)
+        feature_set_1.plot(with_targets=True)
+        if local:
+            fstore.ingest(
+                featureset=feature_set_1,
+                source=first_data,
+                run_config=fstore.RunConfig(local=local),
```

Comment:
I think you reversed the logic here - you want to create a `RunConfig` if *not* running locally. Also, there's not much point in creating it with `local=local` when we know in advance that `local` is `True`.
Why do you need the `if local:` anyway? Why not always running with a `RunConfig` and passing the `local` parameter to it?